aviary/docs/user_guide/hamilton_standard.md

```diff
@@ -0,0 +1,101 @@
+# Hamilton Standard Propulsion Model
+
+In the 1970s, NASA contracted Hamilton Standard to forecast into the future mid-80s to the 90s what they thought advanced propellers would look like. They took existing data and projected into the future. The result is what we call “Hamilton Standard model” today. The Hamilton Standard Documentation is publicly available on NTRS: [19720010354.pdf(https://ntrs.nasa.gov/api/citations/19720010354/downloads/19720010354.pdf) (nasa.gov). You can find the definitions, methodology, and Fortran code in the doc. In Aviary, we implement only one of the options, namely, for a given horsepower, it computes the corresponding thrust.
+
+Below is an XDSM diagram of Hamilton Standard model:
+
+![Hamilton Standard Diagram](images/hamilton_standard.png)
+
+The inputs are grouped in three aspects:
+
+Geometric inputs:
+- Propeller diameter
+- Activity factor per blade (range: 80 to 200, baseline: 150)
+- Number of Blades (range: 2 to 8)
+
+Power inputs:
+- Shaft Horsepower to Propeller (hp)
+- Installation Loss Factor (0 to 1)
+
+Performance inputs:
+- Operating Altitude, ft
+- True Airspeed, knots
+- Propeller Tip Speed (Usually < 800 ft/s) 
```

Comment:
is this maximum or instantaneous?
If it is max as set by a user, make a note of it.
If it is just current from rpm*radius, no change is needed.

Reply:
It is not the maximum tip speed.